external/mmdetection/tests/test_ote_api.py

```diff
@@ -542,14 +542,13 @@ def test_training_yolox(self):
             osp.join('configs', 'custom-object-detection', 'cspdarknet_YOLOX'))
 
     @e2e_pytest_api
-    @pytest.mark.xfail(reason='CVS-83115')
     def test_training_maskrcnn_resnet50(self):
         self.end_to_end(osp.join('configs',
                         'custom-counting-instance-seg', 'resnet50_maskrcnn'),
                         task_type=TaskType.INSTANCE_SEGMENTATION)
 
     @e2e_pytest_api
-    @pytest.mark.xfail(reason='CVS-83116')
+    @pytest.mark.xfail(reason='It randomly cannot pass export_perf_delta_tolerance or pot_perf_delta_tolerance')
```

Comment:
Please use Jira ticket ID. You may want to edit existing one or add another.

Reply:
Fixed